src/rpc/chain.ts

```diff
@@ -997,11 +997,13 @@ export class ChainRpc {
      * Gets pending transactions that have the insertion timestamp within the given range.
      * @param from The lower bound of the insertion timestamp.
      * @param to The upper bound of the insertion timestamp.
+     * @param futureIncluded is designed to include the future transactions in the future queue.
```

Comment:
It still starts with "is". This part will be weird when it's displayed in the API documentation site. (you can see the screenshot below)
Please write either a complete sentence or a noun phrase after `@param futureIncluded`.

![image](https://user-images.githubusercontent.com/9711879/71714777-7ac2bc80-2e52-11ea-9cc9-8f0797765456.png)